samples/mqtt/sender/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/eclipse/paho.golang/paho"
+	"github.com/google/uuid"
+
+	cemqtt "github.com/cloudevents/sdk-go/protocol/mqtt_paho/v2"
+)
+
+const (
+	count = 10
+)
+
+func main() {
+	conn, err := net.Dial("tcp", "127.0.0.1:1883")
+	if err != nil {
+		log.Fatalf("failed to connect to mqtt broker: %s", err.Error())
+	}
+	clientConfig := &paho.ClientConfig{
+		ClientID: "sender-client-id",
+		Conn:     conn,
+	}
+	cp := &paho.Connect{
+		KeepAlive:  30,
+		CleanStart: true,
+	}
+
+	p, err := cemqtt.New(context.TODO(), clientConfig, cp, "test-topic", nil, 0, false)
```

Comment:
nit: create a `ctx` and reuse instead of TODO

Reply:
Done!